gravity-forms/gw-capitilize-submitted-data.php

```diff
@@ -27,8 +27,14 @@ function gw_capitalize_submitted_data( $form ) {
 				$_POST[ $input_key ] = ucwords( strtolower( rgpost( $input_key ) ) );
 			}
 		} else {
-			$input_key           = sprintf( 'input_%s', $field['id'] );
-			$_POST[ $input_key ] = ucwords( strtolower( rgpost( $input_key ) ) );
+			$input_key = sprintf( 'input_%s', $field['id'] );
```

Comment:
I'd add an explicit check for List fields here, just to make the code more clear and to avoid future confusion if issues arise.